### PR TITLE
persistent access_token storage

### PR DESCRIPTION
**Pull request by ivyastrix**
We are now storing access tokens in the DB instead of session storage.

### CODE REVIEW

File: pages/api/exchange-public-token.js [hunk 1]
```diff
@@ -8,7 +9,25 @@ async function exchangePublicToken(req, res) {
...
+        port: Number(process.env.DB_PORT),
+        ssl: {
+            rejectUnauthorized: false,
+            // Add any other SSL configuration options here
+        }
+    });
+
+    console.log("access token" + exchangeResponse.data.access_token);
+
+    const email = "ivy@ivyastrix.com"
```
Comment (hone1er) [inline pages/api/exchange-public-token.js:26]: Got your email hardcoded here

**Review by hone1er — CHANGES_REQUESTED** [review]
Just gotta change that one hard coded email and it looks good

File: pages/api/exchange-public-token.js (hunk 1 above)
Comment (ivyastrix) [reply pages/api/exchange-public-token.js:26]: omg!!!!

Comment (ivyastrix) [reply pages/api/exchange-public-token.js]: k done!!

**Review by hone1er — APPROVED** [review]

**Pull request merged**